msl/equipment/resources/isotech/millik.py

```diff
@@ -0,0 +1,184 @@
+
+        Do not instantiate this class directly. Use the :meth:`~.EquipmentRecord.connect`
+        method to connect to the equipment.
+
+        :param record: A record from an :ref:`equipment-database`.
+        """
+        interface = record.connection.interface
+        if interface == Interface.SOCKET:
+            base = ConnectionSocket
+        elif interface == Interface.SERIAL:
+            base = ConnectionSerial
+        else:
+            raise IsoTechError(f"Unknown interface for connection for {record.connection.interface}")
+
+        dict_ = dict((k, v) for k, v in vars(cls).items() if not k.startswith('__'))
+        type_ = type(cls.__name__, (base,), dict_)
+        instance = type_(record)
+        instance.set_exception_class(IsoTechError)
+
+        instance.rstrip = True
+        instance.read_termination = '\r'
+        instance.write_termination = '\r'
+        instance.write('millik:remote')  # use REMOTE mode to speed up communications
+
+        instance._connected_devices, instance._num_devices, instance._channel_numbers = _find_channel_numbers(instance)
+
+        instance.channel_configuration = {}
+        """A list of configured channel numbers with their measurement mode, range, current, and wiring settings."""
+
+        return instance
+
+    @property
+    def connected_devices(self) -> list[str]:
+        """A list of information about the connected devices (manufacturer, model, serial number, firmware version),
+        e.g. ['Isothermal Technology,millisKanner,21-P2593,2.01', 'Isothermal Technology,milliK,21-P2460,4.0.0'].
+        """
+        # These are the strings that would be returned from each instrument by the *IDN? command
+        return self._connected_devices
+
+    @property
+    def num_devices(self) -> int:
+        """The number of connected devices."""
+        return self._num_devices
+
+    @property
+    def channel_numbers(self) -> list[int]:
+        """A list of available channel numbers, e.g. [1, 2] for a single milliK,
+        or [1, 10, 11, 12, 13, 14, 15, 16, 17] for a milliK connected to a single millisKanner, etc.
+        """
+        return self._channel_numbers
+
+    def configure_resistance_measurement(self, channel: int, meas_range: float, *, norm: bool = True, fourwire: bool = True)\
+            -> None:
+        r"""Configure the milliK to measure resistance for the specified channel.
+
+        :param channel: The channel to configure for resistance measurement.
+        :param meas_range: The measurement range in ohms. Selects from 115 Ohms, 460 Ohms and 500 kOhms.
+        :param norm: The sense current to use for measurement. Defaults to use normal (1 mA) sense current,
+            unless False in which case it uses root2*1 mA to determine self-heating effects.
+            Thermistors always use 2 μA.
+        :param fourwire: The wiring arrangement eg 3 or 4 wire.
+        :return: Returns the internal values set for range, current, and wiring.
```

Comment:
This method now returns `None` so the `:return:` line should be deleted.

Reply:
OK, well spotted!